### PR TITLE
debian: python related changes

### PR DESCRIPTION
**Pull request by tchaikov**
@dmick could you take a look at this PR also? thanks.

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -134,6 +134,14 @@ override_dh_strip:
 override_dh_shlibdeps:
 	dh_shlibdeps -a --exclude=erasure-code --exclude=rados-classes --exclude=compressor
 
+override_dh_python2:
+	for binding in rados cephfs rbd; do                                \
+	  dh_python2 -O--buildsystem=python_distutils -p python-$$binding; \
+	done
+	dh_python2 -p ceph-common
+	dh_python2 -O--buildsystem=python_distutils -p ceph-base
```
Comment (dmick) [inline debian/rules]: why --buildsystem=python_distutils?  and why not for ceph-common?

**Comment by dmick** [comment]
the build failure is unconnected, most likely, and can be fixed with a rebase I'm assuming

**Comment by tchaikov** [comment]
@dmick ping.

File: debian/rules [hunk 2]
```diff
@@ -134,6 +134,14 @@ override_dh_strip:
 override_dh_shlibdeps:
```
Comment (dmick) [inline debian/rules]: the commit message confuses me; if the warning as about running dh_python2 against python-rbd, the code below explicitly does that; did you mean to have a different package name in the warning above?...

File: debian/control [hunk 3]
```diff
@@ -53,7 +53,7 @@ Build-Depends: btrfs-tools,
                pkg-config,
                python (>= 2.6.6-3~),
                python-argparse | libpython2.7-stdlib,
-               python-dev,
+               python-all-dev,
```
Comment (dmick) [inline debian/control]: Should this be ${python:<something>} anyway?

File: debian/rules (hunk 2 above)
Comment (tchaikov) [inline debian/rules]: @dmick sorry for the misleading example, yeah, lemme copy another warning message.

File: debian/control (hunk 3 above)
Comment (tchaikov) [inline debian/control]: no, ${python:`<something>`} subst vars are generated by dh_python*, it by itself is a build dependency listed as "dh-python". so it's like an problem of egg-and-chicken or self-bootstrap. we can hardly let dpkg-buildpackage to figure out what the build-dependency is before the build-dependencies are ready.

**Comment by tchaikov** [comment]
@dmick thanks for reviewing, fixed the commit message, rebased against master, and repushed.

File: debian/control (hunk 3 above)
Comment (dmick) [inline debian/control]: got it.

**Pull request merged**